linux_os/guide/services/ssh/ssh_server/sshd_use_strong_macs/oval/shared.xml

```diff
@@ -1 +1,71 @@
-{{{ oval_sshd_config(parameter="MACs", value="((hmac-sha2-512-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|umac-128-etm@openssh\.com|hmac-sha2-512|hmac-sha2-256|hmac-ripemd160),?)+") }}}
+<def-group>
+  <definition class="compliance" id="sshd_use_strong_macs" version="1">
```

Comment:
Couldn't find a nice way to transform the comma separated lilst with pipe separated list so can directly use the macro, and didn't felt right to transform the macro that is used in other places.

Reply:
I think with jinja you can just call `replace(",", "|")`

Reply:
> I think with jinja you can just call `replace(",", "|")`

nvm, I think it won't work: https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.replace

Reply:
maybe with join:
https://jinja.palletsprojects.com/en/2.10.x/templates/#join

Reply:
Yes I did but just getting a bar separated list doesn't get me much further since there is need to do much more changes, which I am afraid cause much more noise in the macros, so decided it doesn't worth at this stage